src/deno_dir.rs

```diff
@@ -155,9 +155,9 @@ impl DenoDir {
       self.resolve_module(module_specifier, containing_file)?;
 
     debug!(
-            "code_fetch. module_name = {} module_specifier = {} containing_file = {} filename = {}",
-            module_name, module_specifier, containing_file, filename
-        );
+      "code_fetch. module_name = {} module_specifier = {} containing_file = {} filename = {}",
+      module_name, module_specifier, containing_file, filename
+    );
```

Comment:
Have you run ./tools/format.py ? This has been automatically formatted this way

Reply:
@ry Sorry I did not know. deleted this commit.